notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<p>Batch Serving -&gt; Batch serving</p><p> </p><p>no future tense - drop the word 'will'</p><p> </p><p>Title needs to be a verb phrase (currently a noun phrase)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/486/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>